### ISSUE TITLE
Health check

### ISSUE THREAD

**Issue by Emily-Jiang** [eclipse/microprofile-service-mesh#9]
MP Health check should be used by Kube's health check as MP Health check is more accurate.

**Comment by Emily-Jiang** [comment]
The Health check is useful for Istio Envoy as it needs to query the pod health status.

**Comment by kenfinnigan** [comment]
Though I agree the MP Health Check can certainly be used by Kube, Envoy, and other infrastructure pieces, I don't see there being a need for any "integration" between them.

It's more a case of Kube, Envoy, etc, utilizing the MP Health Check that is provided

**Comment by Emily-Jiang** [comment]
@kenfinnigan I agree with you. This issue will result into a statement not any real outstanding api changes.

**Comment by vadimeisenbergibm** [comment]
The FAQ item in Istio regarding health checks with mTLS https://istio.io/help/faq/security/#k8s-health-checks.
